src/front/app/reducer/app.js

```diff
@@ -1 +1,6 @@
-export default () => ({});
+import { combineReducers } from 'redux';
+import challengeTopics from './challengeTopics';
+
+export default combineReducers({
```

Comment:
combine means that you had 2 and more reducers you want to combine, here you got only one